src/mpid/ch4/netmod/ofi/ofi_win.h

```diff
@@ -1240,6 +1240,10 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_win_free_hook(MPIR_Win * win)
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_WIN_FREE_HOOK);
 
     if (MPIDI_OFI_ENABLE_RMA) {
+        /* ofi window is not initialized */
+        if (!MPIDI_OFI_WIN(win).win_id)
```

Comment:
Is it safe to assume that valid `win_id` is always non-zero? (even after applying #3493?)

Reply:
It looks like the new `win_id` can be zero. Do you have any suggestion to figure out whether ofi_win is initialized ? I could not find any existing parameter that we can reuse.

An alternate option would be to add an `init_status` in `MPIDI_Devwin_t` to indicate whether the shm|nm init hook has been called. What do you think ?

Reply:
I deleted this commit in the last push to keep this PR simple. I do not think it is an issue that we have to address in this PR.

Reply:
Okay.